config/jobs/kubernetes/sig-release/release-branch-jobs/1.15.yaml

```diff
@@ -979,7 +979,7 @@ presubmits:
         - runner.sh
         env:
         - name: WHAT
-          value: vendor vendor-licenses
+          value: external-dependencies-version vendor vendor-licenses
```

Comment:
Are we still supporting 1.15 jobs?

Reply:
@cblecker -- For now, yes. We usually blow them away for beta.0, which is right around the corner.

/hold cancel